### PR TITLE
Link to latest in scenario name

### PR DESCRIPTION
**Pull request by okurz**
Screenshot:
openqa_link_to_latest_in_previous

### CODE REVIEW

File: templates/test/previous.html.ep [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div id="scenario">
-    <div class="h5">Results for <i><%= $scenario %></i></div>
+    <div class="h5">Results for <i><%= link_to($scenario => url_for ('latest')->query($job->scenario_hash)) %></i></div>
```
Comment (aaannz) [inline templates/test/previous.html.ep]: Tried it and it's confusing for me. I would prefer something like:
```
<div class="h5">Results for <i><%= $scenario %></i> (<%= link_to('latest results for this scenario' => url_for('latest')->query($job->scenario_hash)) %>)</div>
```

Comment (okurz) [inline templates/test/previous.html.ep]: On Friday 26 August 2016 07:26:07 Ondřej Holeček wrote:
> > @@ -1,5 +1,5 @@
> > 
> >  <div id="scenario">
> > 
> > -    <div class="h5">Results for <i><%= $scenario %></i></div>
> > +    <div class="h5">Results for <i><%= link_to($scenario => url_for
> > ('latest')->query($job->scenario_hash)) %></i></div>
> Tried it and it's confusing for me. I would prefer something like:
> ```
> <div class="h5">Results for <i><%= $scenario %></i> (<%= link_to('latest
> results for this scenario' =>
> url_for('latest')->query($job->scenario_hash)) %>)</div> ```

sure, I wouldn't mind

**Comment by okurz** [comment]
updated:
 * made link more explicit as suggested by @aaannz 
 * added screenshot to description


@coolo your opinion?

File: templates/test/previous.html.ep (hunk 1 above)
Comment (coolo) [inline templates/test/previous.html.ep]: latest results implies it's multiple results, but it's just one -> 'latest job' then

**Comment by coolo** [comment]
other than the wording I'm fine with the change

**Comment by okurz** [comment]
@aaannz the trim_whitespace failed me in https://travis-ci.org/os-autoinst/openQA/builds/156827974

**Comment by okurz** [comment]
@coolo, @aaannz updated wording, just using regex match for the test to not replicate the full text content and confuse `trim_whitespace`. Please merge and deploy, should make anyone happy asking in bugs: "Does this still happen?" ;-)

**Comment by aaannz** [comment]
I don't know, `trim_whitespace` replace multiple spaces by one and removes spaces at the beginning and end of the string. From my limited perspective it seems to me you wanted to remove one space near () inside of the string.

**Comment by aaannz** [comment]
btw. LGTM :)

**Comment by okurz** [comment]
@coolo was fine with the change, name is fixed, want to merge?

**Pull request merged**